layout/_partial/head.ejs

```diff
@@ -4,6 +4,7 @@
         © Material Theme
         https://github.com/viosey/hexo-theme-material
         Version: 1.3.4 -->
+    <script>window.materialVersion = "1.3.4"</script>
```

Comment:
Maybe write version at <html materialVersion="1.3.4"> ?
I think this will cause render lagging...